### PR TITLE
OM-786 | Add accessibility docs

### PR DESCRIPTION
**Pull request by tuovinensanttu**
Added accessibility statement.

### CODE REVIEW

File: src/accessibilityStatement/AccessibilityStatement.module.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
+.pageWrapper {
+    min-height: 100vh;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+}
+
+.container {
+    margin: 20px 10px 0;
```
Comment (Jokinen) [inline src/accessibilityStatement/AccessibilityStatement.module.css]: I am thinking if we should not start using spacing variables from HDS now that they are relatively stable?

File: src/accessibilityStatement/AccessibilityStatement.module.css [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+.innerWrapper h1 {
+    font-size: var(--fontsize-h-3);
+}
+
+.innerWrapper h2 {
+    margin-top: 25px;
+    font-size: var(--fontsize-h-4);
+}
+
+@media (min-width: 800px) {
```
Comment (Jokinen) [inline src/accessibilityStatement/AccessibilityStatement.module.css]: There are different breakpoints in the app, but this is the only one of `800px`. The closes value currently in use would be `768px`. I don't know if it makes a huge difference for this case.

File: src/accessibilityStatement/AccessibilityStatement.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+import Header from '../common/header/Header';
+import Footer from '../common/footer/Footer';
+import AccessibilityStatementFi from './AccessibilityStatementFi';
+import AccessibilityStatementSv from './AccessibilityStatementSv';
+import AccessibilityStatementEn from './AccessibilityStatementEn';
+
+function AccessibilityStatement() {
+  const { i18n } = useTranslation();
+  const selectStatement = () => {
+    const lang = i18n.languages[0];
```
Comment (Jokinen) [inline src/accessibilityStatement/AccessibilityStatement.tsx]: Might this language value also sometimes contain the locale information? If I remember correctly you configured locales with both youth-membership and profile.

File: src/accessibilityStatement/AccessibilityStatementFi.tsx [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+        palautelomakkeella{' '}
+        <a href="https://www.hel.fi/palaute">www.hel.fi/palaute</a>.
+        Tiedusteluun pyritään vastaamaan kohtuullisessa ajassa.
+      </p>
+
+      <h2>Palaute ja yhteystiedot</h2>
+      <p>
+        Kaupunginkanslia, Helsinki
+        <br />
+        +358 9 310 1641
```
Comment (Jokinen) [inline src/accessibilityStatement/AccessibilityStatementFi.tsx]: A `tel:` link is possible here. But maybe it has been omitted for a specific reason?

**Review by Jokinen — APPROVED** [review]
Looks good to me 👍 A few comments.

File: src/accessibilityStatement/AccessibilityStatement.module.css (hunk 1 above)
Comment (tuovinensanttu) [reply src/accessibilityStatement/AccessibilityStatement.module.css]: Sure, I'll change these to use hds spacing.

File: src/accessibilityStatement/AccessibilityStatement.module.css (hunk 2 above)
Comment (tuovinensanttu) [reply src/accessibilityStatement/AccessibilityStatement.module.css]: I'll change this to `768` to keep things consistent.

File: src/accessibilityStatement/AccessibilityStatement.tsx (hunk 3 above)
Comment (tuovinensanttu) [reply src/accessibilityStatement/AccessibilityStatement.tsx]: You are totally right, nice catch 👍

File: src/accessibilityStatement/AccessibilityStatementFi.tsx (hunk 4 above)
Comment (tuovinensanttu) [reply src/accessibilityStatement/AccessibilityStatementFi.tsx]: Added `tel:` link.

**Review by badrange — APPROVED** [review]
👍

**Pull request merged**